apps/src/code-studio/initApp/renderPrivacyProfanityViolating.js

```diff
@@ -0,0 +1,24 @@
+/* global dashboard */
+import React from 'react';
+import ReactDOM from 'react-dom';
+var AbuseExclamation = require('../components/abuse_exclamation');
+var showProjectAdmin = require('../showProjectAdmin');
```

Comment:
any reason you're requiring instead of importing these second two? at the least you could import as a const instead of a var